Source/Android/app/src/main/java/org/dolphinemu/dolphinemu/ui/settings/SettingsFragmentPresenter.java

```diff
@@ -277,7 +279,7 @@ private void addHackSettings(ArrayList<SettingsItem> sl)
 
 		sl.add(new HeaderSetting(null, null, R.string.embedded_frame_buffer, 0));
 		sl.add(new CheckBoxSetting(SettingsFile.KEY_SKIP_EFB, SettingsFile.SECTION_GFX_HACKS, R.string.skip_efb_access, R.string.skip_efb_access_descrip, false, skipEFB));
```

Comment:
Nope, the inverted setting gets passed to the checkbox, and the checkbox saves the setting inverted with a special case in the onClick event.